platforms/bukkit/src/main/java/com/dfsek/terra/bukkit/util/VersionUtil.java

```diff
@@ -40,10 +40,6 @@ public static SpigotVersionInfo getSpigotVersionInfo() {
         private final boolean spigot;
         private final boolean paper;
         private final boolean mohist;
-        private final boolean airplane;
-        private final boolean tuinity;
-        private final boolean purpur;
```

Comment:
Given that tuinity is now merging with paper and yatopia is gone, I'm fine with this change.

Though, we might want to keep purpur (?)

Mainly my reason for including the other forks was because "there's already a few forks, so might as well include the other major ones too"

Reply:
Why manually get the brand from a pre-made list of brand names when Bukkit has API for just that, `Bukkit.getServer().getName()`